vncviewer/parameters.cxx

```diff
@@ -101,10 +101,13 @@ IntParameter qualityLevel("QualityLevel",
                           8);
 
 BoolParameter maximize("Maximize", "Maximize viewer window", false);
-BoolParameter fullScreen("FullScreen", "Full screen mode", false);
+BoolParameter fullScreen("FullScreen", "Enable full screen", false);
+StringParameter fullScreenMode("FullScreenMode", "Specify what mode to use when full screen is enabled. "
+                                                 "Should be either Current or All",
```

Comment:
It should probably be explicit what "Current" and "All" are referring to. So instead of "mode", perhaps write "Specify which monitors to use when...".